frontend/packages/client/src/App.js

```diff
@@ -31,13 +32,15 @@ function App() {
       <QueryClientProvider client={queryClient}>
         <Web3Provider network={process.env.REACT_APP_FLOW_ENV}>
           <Router>
-            <NotificationServiceProvider>
-              <NotificationModalProvider>
-                <ErrorHandler>
-                  <AppPages />
-                </ErrorHandler>
-              </NotificationModalProvider>
-            </NotificationServiceProvider>
+            <ChakraProvider>
+              <NotificationServiceProvider>
+                <NotificationModalProvider>
+                  <ErrorHandler>
+                    <AppPages />
+                  </ErrorHandler>
+                </NotificationModalProvider>
+              </NotificationServiceProvider>
+            </ChakraProvider>
```

Comment:
our app.js is starting to look like this but its OK 

![image](https://user-images.githubusercontent.com/15314629/196548353-380db83b-afb3-4a61-9818-d3c84bb1ba7e.png)